windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -19,6 +19,18 @@ You must have a subscription to receive support for the Red Hat WMCO. Deploying
 
 include::modules/wmco-prerequisites.adoc[leveloffset=+1]
 
+[id="wmco-2-0-3"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.3
+
+Issued:2021-07-28
+
+The WMCO 2.0.3 is now available with bug fixes. The components of the WMCO were released in link:https://errata.devel.redhat.com/docs/show/79455[RHBA-2021:79455-02].
```

Comment:
as mentioned in Slack, try to get the Live ID for this so you don't have to chase it later :)